docs/source/staking/running_a_worker.rst

```diff
@@ -227,6 +227,36 @@ Replace the following values with your own:
    * ``<GWEI>`` - The maximum price of gas to spend on commitment transactions
 
 
+The configuration settings will be stored in an ursula configuration file.
+
+.. important::
+
+    The default configuration file is ``ursula.json``. If there is an existing default configuration file, a new
+    Worker configuration file suffixed by the first 8 characters of the node's signing public key
+    e.g. ``ursula-0216ad10.json``. Since this file would not be the default, subsequent ``ursula`` CLI commands
+    should use the ``--config-file <FILEPATH>`` option to specify the non-default filepath of the Worker
+    configuration file.
+
+
+.. note::
+
+    All worker configuration values can be changed using the `config` command
+    (Note that the worker must be restarted for new changes to take effect):
```

Comment:
I think this note deserves it's own note section.

Reply:
✔️